src/Umbraco.Web/PropertyEditors/ValueConverters/MediaCropperPropertyConverter.cs

```diff
@@ -0,0 +1,270 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MediaPickerPropertyConverter.cs" company="Umbraco">
+//   Umbraco
+// </copyright>
+// <summary>
+//  The media picker 2 value converter
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Umbraco.Core;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Models;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Core.PropertyEditors;
+using Umbraco.Core.Services;
+using Umbraco.Web.Extensions;
+using Umbraco.Web.Models;
+
+namespace Umbraco.Web.PropertyEditors.ValueConverters
+{
+    /// <summary>
+    /// The media picker property value converter.
+    /// </summary>
+    [DefaultPropertyValueConverter]
```

Comment:
@skttl Try doing what the `RelatedLinksPropertyConverter` value-converter does, by adding in the "shadow converter"
e.g. https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web/PropertyEditors/ValueConverters/RelatedLinksEditorValueConvertor.cs#L28

`[DefaultPropertyValueConverter(typeof(JsonValueConverter))]`

I'm not entirely sure what "shadow" means in this context, but if the related-links converter is doing it, then it's worth a try?

Reply:
That worked! Thank you very much :)

Reply:
The default `JsonValueConverter` is automatically registered for all editors that store their value as JSON, but because the `MediaCropperPropertyConverter` is also a default converter, Umbraco doesn't know which one to use...

When writing custom (non-default) PVCers, you don't have to do this BTW, since they always override (shadow) default converters 👍